shell/src/main/java/alluxio/cli/bundler/command/CollectEnvCommand.java

```diff
@@ -38,33 +38,33 @@ public CollectEnvCommand(FileSystemContext fsContext) {
   @Override
   protected void registerCommands() {
     registerCommand("Alluxio ps",
-            new ShellCommand(new String[]{"bash", "-c", "'ps -ef | grep alluxio'"}), null);
+            new ShellCommand(new String[]{"bash", "-c", "ps", "-ef", "| grep alluxio"}), null);
```

Comment:
what's the logic for this split?

Reply:
Doing this just because `bash -c` seems to use the next string slice to look for. the executable. It doesn't seem to work with the whole command on my Mac somehow...